tomviz/python/GaussianFilterTiltSeries.json

```diff
@@ -0,0 +1,14 @@
+{
+  "name" : "GaussianFilter",
+  "label" : "Gaussian Filter",
```

Comment:
Turns out the empty UI above is caused by wrong format in Mac's TextEdit (default for .json file). For example, the double quote character in TextEdit becomes 
<img width="200" alt="screen shot 2016-10-26 at 12 37 11 pm" src="https://cloud.githubusercontent.com/assets/13088588/19735367/43dd0f50-9b79-11e6-91a6-cfcdd66d37be.png">

Note to self: use Xcode to edit .json file.